Apache-Beam-Jobs.md

```diff
@@ -433,12 +419,52 @@ def run(self):
         | 'Sum values' >> beam.CombineGlobally(sum)
     )
 
-    return state_count_sum_pcoll | beam.Map(job_run_result.JobRunResult.as_stdout)
+    return (
+        state_count_sum_pcoll
+        | 'Map as stdout' >> beam.Map(job_run_result.JobRunResult.as_stdout)
+    )
 ```
 
-The method maps every element in a `PCollection` into a `JobRunResult` with the stringified-value as its `stdout`.
+The method maps every element in a `PCollection` to a `JobRunResult` with their stringified-values as `stdout`.
```

Comment:
Missing "its" before "stdout", perhaps? (I'm not really sure what the sentence as-written means.)

Reply:
Done.